macros/src/main/scala/gestalt/macros/Optional.scala

```diff
@@ -28,11 +28,14 @@ final class Optional[+A >: Null](val value: A) extends AnyVal {
         Select(tempIdent, "value")
     }
 
-    q"""
+    {
+      import scala.gestalt.options.unsafe
```

Comment:
Oh, right, `NewInstance` no longer can give names to things. I removed the `Unsafe`.
But this block is still __unsafe__ because of the `TypeIdent.apply("Optional")` generated by the quasiquotes.